### PR TITLE
Add MSG_SBAS_RAW

### PR DESCRIPTION
**Pull request by pmiettinen**
https://paper.dropbox.com/doc/SBAS-Correction-usage-in-Starling-lj63ycGD0rhdeQ7zt8KTl#:uid=034412587439856993266169&h2=SBP-message

### CODE REVIEW

**Comment by pmiettinen** [comment]
Waiting for confirmation for the actual spec.

**Comment by kimvaarala** [comment]
@anth-cole Was there an approval for the proposed SBAS RAW data message (as suggested here: https://paper.dropbox.com/doc/SBAS-Correction-usage-in-Starling-lj63ycGD0rhdeQ7zt8KTl#:uid=034412587439856993266169&h2=SBP-message).

This PR is the implementation of that message.

File: c/include/libsbp/sbas.h [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2015 Swift Navigation Inc.
```
Comment (anth-cole) [inline c/include/libsbp/sbas.h]: change year to 2018

File: c/include/libsbp/sbas.h [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ * Contact: Fergus Noble <fergus@swift-nav.com>
```
Comment (anth-cole) [inline c/include/libsbp/sbas.h]: use the developer@swiftnav.com email please

File: c/include/libsbp/sbas.h [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+/** Raw SBAS data
+ *
+ * This message is sent once per second per SBAS satellite. ME checks the
+ * parity of the data block and sends only blocks that pass the check.
+ */
+#define SBP_MSG_SBAS_RAW 0x7777
+typedef struct __attribute__((packed)) {
+  sbp_gnss_signal_t sid;             /**< GNSS signal identifier. */
+  u32 tow;             /**< GPS time-of-week at the start of the data block. [ms] */
+  u8 message_type;    /**< SBAS message type (0-63) */
```
Comment (anth-cole) [inline c/include/libsbp/sbas.h]: Why do we need to transmit message_type? Won't the message ID in the data field give this information?

Comment (nsirola) [reply c/include/libsbp/sbas.h]: This is the SBAS Message Type, which is outside the 212 bit data field

Comment (anth-cole) [reply c/include/libsbp/sbas.h]: ok, got it thanks

**Comment by anth-cole** [comment]
@pmiettinen if you can make those couple of nit changes, this is good to merge

File: c/include/libsbp/sbas.h (hunk 1 above)
Comment (pmiettinen) [reply c/include/libsbp/sbas.h]: These come from the template. I think it's better to update these in separate PR and preferably by the repository POC.

Comment (pmiettinen) [reply c/include/libsbp/sbas.h]: In other words I'd ping @denniszollo about this.

Comment (anth-cole) [reply c/include/libsbp/sbas.h]: fair enough then - they were only nits - looks good

**Pull request merged**